modules/manage/pages/recovery-mode.adoc

```diff
@@ -0,0 +1,218 @@
+= Recovery Mode
+:description: Recovery mode starts Redpanda in "metadata only" mode and disables partitions so you can repair a failed cluster.
+
+Recovery mode allows you to start Redpanda and form a stable cluster when it fails due to a system crash, OOM errors, or other issues that make a normal startup infeasible. 
```

Comment:
```suggestion
Recovery mode allows you to repair and restore a failed cluster that cannot start normally due to issues such as a system crash or out-of-memory (OOM) errors. In recovery mode, Redpanda limits functionality to cluster configuration changes and other manual administrative actions so that you can repair the cluster. 
```